### PR TITLE
Fix missing permission `--allow-sys`

### PR DESCRIPTION
**Pull request by ericlery**
Fix: #223 

See: https://github.com/denoland/deployctl/blame/ad8328ef14088b26a27762c2bf41b47e5b8073c4/src/utils/api.ts#L19

### CODE REVIEW

File: deno.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     }
   },
   "tasks": {
-    "test": "deno test -A --unstable tests/ src/",
+    "test": "deno test -A --unstable --shuffle --parallel tests/ src/",
```
Comment (magurotuna) [inline deno.json:13]: Question: is this relevant to the issue?

**Review by magurotuna — COMMENTED** [review]
Thanks for reporting and fixing, generally looking good. Commented one question though

File: deno.json (hunk 1 above)
Comment (ericlery) [reply deno.json:13]: It's not, therefore I can split it in another branch and PR

Comment (ericlery) [reply deno.json]: It is not, it was just an easy add-in. I have now pulled out the commit containing this addition.

**Review by magurotuna — APPROVED** [review]
LGTM!
